src/components/Locale/Locale.tsx

```diff
@@ -205,15 +218,14 @@ function getKeyValueJson(messages: LocaleMessages): Record<string, string> {
   }
 }
 
-const defaultLocale = Locale.EN;
-
 const LocaleProvider: React.FC = ({ children }) => {
-  // For now locale can be set here
-  const locale = Locale.EN;
+  const {
+    preferences: { locale },
+  } = usePreferences();
 
   return (
     <IntlProvider
-      defaultLocale={defaultLocale}
+      defaultLocale={Locale.EN}
```

Comment:
I wonder if it isn't something that should be passed from env

Reply:
I can be wrong but isn't `defaultLocale` setting used for fallbacks? In our case default messages, which are set in message descriptors, are in English, and basing on that I deduced that we can leave it hardcoded this way.
Default language set in env is a great suggestion, but maybe we should rather use it for the default value for `PreferencesProvider`?

Reply:
I just committed my proposition for passing `defaultLocale` from env to `PreferencesProvider`. @dominik-zeglen and @krzysztofwolski please take a look